packages/network/src/fetch.ts

```diff
@@ -41,10 +42,19 @@ export interface BuildFetchOptions {
    * themselves.
    */
   disableMessage?: string;
+
   /**
    * override the default fetch implementation
    */
-  fetch: typeof fetch;
+  fetch?: typeof fetch;
```

Comment:
This should have always been optional, you should be able to provide the other options without also being forced to override `fetch`